### ISSUE TITLE
Character Encoding Error in url

### ISSUE THREAD

**Issue by vinodsingh581** [odoo/odoo#63563]
Odoo 13.0

Steps to reproduce:

I have checked this only for the Arabic language, please check

1. Go to the website and load the Arabic language.
2. Then hit any of the URL from the website in the Arabic language ex: /ar/اختبار

Current behavior:

Give us an internal server error:  UnicodeEncodeError: 'latin-1' codec can't encode characters in position

Expected behavior:

404 not found

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by vinodsingh581** [comment]
Thanks for your reply, Please see the attached video
[error.webm.tar.gz](https://github.com/odoo/odoo/files/5714652/error.webm.tar.gz)

**Comment by rdeodoo** [comment]
@vinodsingh581 Hi, any chance you can upload the video, eg on google drive?

**Comment by vinodsingh581** [comment]
Hi @rdeodoo 
Get the error video from the google drive link
https://drive.google.com/file/d/1FgAxmpGnnH-hyr6mk4TYHGtpNnLZOXFN/view?usp=sharing

**Comment by ISKO0** [comment]
Did You Find Any Solution

**Comment by ISKO0** [comment]
@rdeodoo

**Comment by bso-odoo** [comment]
> Did You Find Any Solution

Not really: there is no error anymore - but this is because all non latin-1 characters are now removed from URLs. In turn, empty URLs then become `-1`, `-2`... to differentiate them.
Forcing a pre-encoded URL is not supported either, and leads to a 404 if you try to access it.
I log it so that this feature gets a priority assigned.
